tactics/btermdn.ml

```diff
@@ -32,6 +31,30 @@ let compare_term_label t1 t2 = match t1, t2 with
 
 type 'res lookup_res = 'res Dn.lookup_res = Label of 'res | Nothing | Everything
 
+let eta_reduce = Reductionops.shrink_eta
```

Comment:
?

Reply:
Shrink_eta is a weird name and there is the similarly named function `eta_reduce_pat` below.

Reply:
We can rename shrink_eta if you don't like it

Reply:
Fine by me, but that would be unrelated to this PR.